Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```
@@ -119,7 +119,7 @@
   - type: InteractionOutline
   - type: MovementSpeedModifier
     baseWalkSpeed: 1
-    baseSprintSpeed: 2
+    baseSprintSpeed: 3.9 #sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)